src/shared/utilities/disposableFiles.ts

```diff
@@ -26,6 +26,10 @@ export class DisposableFiles implements vscode.Disposable {
         return this
     }
 
+    public isDisposed(): boolean {
+        return !!this._disposed
```

Comment:
Is `!!` necessary?

Reply:
to convert to an explicit `boolean` ?

Reply:
I don't understand, the declaration is ` private _disposed: boolean = false `

Reply:
oh haha